src/app/zap-templates/common/ClusterTestGeneration.js

```diff
@@ -202,19 +203,37 @@ function parse(filename)
   return yaml;
 }
 
-// Templates Internal Utils
-
 function printErrorAndExit(context, msg)
 {
   console.log(context.testName, ': ', context.label);
   console.log(msg);
   process.exit(1);
 }
 
+function getClusters()
+{
+  return Clusters.getClusters().then(clusters => {
+    const content = JSON.parse(JSON.stringify(clusters));
```

Comment:
This is worth a comment explaining why we need the JSON bits.  I assume the issue is we want to not modify the list because something else owns the list?  Do we need to do the deep-cloning, or could we just `then(clusters => clusters.concat(GlobalCommands))`?

Reply:
I have added a comment. And the root issue as you guess is that the list is a "reference" to the list generated by `ClustersHelper.js`. I end up using `concat` as it is enough, but at some point I guess it will be better to return a cloned array from the API as it will avoid this type of issues.